### DISCUSSION TITLE
Monitor the maximum pressure and temperature during the calculation

### DISCUSSION THREAD

**Discussion by czc-zju** [AMReX-Combustion/PeleLMeX#449] [Q&A] [answered]
Thank you in advance. I would like to ask how to monitor the maximum pressure and temperature during the calculation or in log. I know that when **_nodal_proj.verbose = 2_**, it seems like some information can be viewed, but the pressure and temperature is not inclued.

**Comment by d-montgomery** [comment, marked as answer]
You can use the [temporals functionality](https://amrex-combustion.github.io/PeleLMeX/manual/html/LMeXControls.html#run-time-diagnostics) to write extremas to file at desired intervals.  Simply add `peleLM.do_temporals = 1` and `peleLM.do_extremas = 1`, along with a desired write interval `peleLM.temporal_int = 10` to your input file.  The max pressure will be under the `max_RhoRT` column.  For an example see the [FlameSheet](https://github.com/dmontgomeryNREL/PeleLMeX/blob/54942ab2d82ac6dad1a6b323df3db251c1cdf8f1/Exec/RegTests/FlameSheet/input.2d-regt#L40) regular test.

**Reply by czc-zju** [reply]
Oh,  I get it now. Thanks a lot!
